src/view/selection.js

```diff
@@ -41,29 +41,41 @@ export default class Selection {
 	 *
 	 *		// Creates selection at the given range.
 	 *		const range = new Range( start, end );
-	 *		const selection = new Selection( range, isBackwardSelection );
+	 *		const selection = new Selection( range, { backward, fake, label } );
```

Comment:
I don't think we should create a fake selection in the all samples. Keep it simple and use `fake` and `label` only in the sample of the fake selection.